app/forms/support_interface/application_forms/pick_course_form.rb

```diff
@@ -0,0 +1,58 @@
+module SupportInterface
+  module ApplicationForms
+    class PickCourseForm
+      include ActiveModel::Model
+      include ValidationUtils
+
+      attr_accessor :course_code, :application_form_id, :candidate_id, :course_option_id
+
+      validates :course_option_id, presence: true
+      validates :application_form_id, presence: true
+      validates :course_code, presence: true
+
+      RadioOption = Struct.new(:course_option_id, :course_name, :course_code, :site_name)
```

Comment:
Do you think we should also have provider_name for clarity? I have come across a course code that is 'Primary' for two different providers? It might be overkill though since they won't share a site.

Reply:
Kind fo just talked myself out of it haha